src/renderer/Settings.tsx

```diff
@@ -313,7 +324,7 @@ const Settings: React.FC<SettingsProps> = function ({ open, onClose }: SettingsP
 			</div>
 			<div className='settings-alert' style={{ display: unsaved ? 'flex' : 'none' }}>
 				<span>
-					Exit to apply changes
+					Close Settings to apply changes
```

Comment:
`Exit` implies they need to close the app entirely, and not close the Settings which seemed to work based on my testing. This change can easily be undone if my understanding of the behavior is incorrect.